README.md

```diff
@@ -89,22 +89,21 @@ Note: `SpinnakerAccount` CRD is optional.
 ### Step 2: Install Operator
 
 There are two modes for the operator:
-- **Basic mode** installs Spinnaker in a single namespace without validating admission webhook.
+- **Basic mode** installs Spinnaker into the same namespace as the Spinnaker Opertor without validating admission webhook.
 - **Cluster mode** works across namespaces and requires a `ClusterRole` to perform validation.
 
 The main difference between the two modes is that basic only requires a `Role` (vs a `ClusterRole`) and has no validating webhook.
 
-Once installed, you should see a new deployment representing the operator. The operator watches for changes to the `SpinnakerService` objects. You can check on the status of the operator using `kubectl`.
+Once installed, you should see a new deployment representing the Operator. The Operator watches for changes to the `SpinnakerService` objects. You can check on the status of the Operator using `kubectl`.
 
 #### Basic install (no validating webhook)
+Decide what namespace you want to use for Operator and create that namespace. We suggest `spinnaker-operator`.
```

Comment:
It would make more sense to me to use `spinnaker` as the namespace. That's the main thing the namespace contains.

Reply:
Note, we actually have MOST of our automation setup to use "spinnaker" as where spinnaker itself is deployed to and spinnaker-operator as where operator is deployed to.  Should spinnaker operator be in "spinnaker" and there be a different standard for spinnaker itself?